laghos.cpp

```diff
@@ -287,7 +287,8 @@ int main(int argc, char *argv[])
         MFEM_VERIFY(numWindows > 0, "");
         if (rom_online || rom_restore)
         {
-            const int err = ReadTimeWindowParameters(numWindows, twpfile, twep, twparam, myid == 0);
+            double sFactor[]  = {2.0, 20.0, 2.0};
```

Comment:
Should we add 3 input parameters for these factors, instead of hard-coding them? These values could be the defaults in case the user does not specify them.

Reply:
good idea! added.